CONTRIBUTING.md

```diff
@@ -39,7 +39,10 @@ Keeping the `master` releasable means that changes merged to it need to be:
 1. Clone the repository.
 2. Run `yarn`. (Installs dependencies and links packages in the workspace.)
 3. Run `git config commit.template .github/.COMMIT_TEMPLATE` (Sets you up with our commit message template)
-4. Run `yarn start` in the root folder. (Start watching and automatically re-building packages when there are new changes.)
+4. Run `yarn build` to build all packages before start watching.
+5. Run `yarn start` in the root folder. (Start watching and automatically re-building packages when there are new changes.)
+6. Open another terminal to test the `expo-cli`, then run `node expo-cli/packages/expo-cli/bin/expo.js init` outside the `expo-cli` repository to create an test repository, if everything is correct, it should create the repository `my-app`.
```

Comment:
this line could be a bit misleading because you would need to include the full path to the expo-cli directory for it to work

Reply:
Maybe saying to open the terminal at the same top level folder that `expo-cli` is located could be less confusing? "6. Open another terminal at the same top level folder that `expo-cli` repository is located, then run `node ~/code/expo-cli/packages/expo-cli/bin/expo.js init`, if everything is correct, it should create the repository `my-app`."